lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2638,7 +2638,7 @@ public function get_root_layout_rules( $selector, $block_metadata ) {
 			$css         .= '--wp--style--global--wide-size: ' . $wide_size . ';';
 		}
 
-		$css .= '}';
```

Comment:
There were a bunch of places in the tests where this was different from core, so I figured I'd just add the space to make them the same.